src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -0,0 +1,54 @@
+package nextstep.subway.line.ui;
+
+import java.net.URI;
+import java.util.List;
+import nextstep.subway.line.application.LineService;
+import nextstep.subway.line.dto.LineRequest;
+import nextstep.subway.line.dto.LineResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
@RequestMapping 활용해줘도 좋을 것 같네요
```suggestion
@RestController
@RequestMapping("/lines")
```